backend/game_server/lib/helpers/vector.ex

```diff
@@ -0,0 +1,41 @@
+defmodule Helpers.NotVector do defexception message: "One of arguments is not Vector" end
```

Comment:
Change you laptop! :P

I can make limitation in sublime same as is in Github so I will see when functions expand.